src/routes/cart.route.js

```diff
@@ -0,0 +1,10 @@
+import express from "express";
+import CartController from "../controllers/cartController";
+import {verifyBuyer} from "../middlewares/authenticate";  
+
+const cartRoute = express.Router();
+
+cartRoute.post("/add/:productId",verifyBuyer,CartController.addItem );
+cartRoute.delete("/clearCart/:id",verifyBuyer,CartController.clearCart );
```

Comment:
Use hyphens to separate words in the endpoint names makes them more readable and understandable, especially when there are multiple words involved.